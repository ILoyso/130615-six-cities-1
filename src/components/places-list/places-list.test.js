import React from 'react';
import renderer from 'react-test-renderer';
import {MemoryRouter} from 'react-router-dom';

import PlacesList from './places-list.jsx';

const placeMock = [
  {
    id: 1,
    img: `apartment-01.jpg`,
    isPremium: true,
    price: 120,
    rating: 93,
    title: `Beautiful & luxurious apartment at great location`,
    type: `apartment`
  }
];

it(`PlacesList correctly renders`, () => {
  const placesList = renderer
    .create(<MemoryRouter>
      <PlacesList
        places={placeMock}
        setActiveItem={jest.fn()}
      />
    </MemoryRouter>)
    .toJSON();

  expect(placesList).toMatchSnapshot();
});
