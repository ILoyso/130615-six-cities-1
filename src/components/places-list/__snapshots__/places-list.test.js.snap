// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlacesList correctly renders 1`] = `null`;
